web/app.js

```diff
@@ -1525,6 +1525,11 @@ const PDFViewerApplication = {
               this.pdfViewer.currentScale = value;
             }
             break;
+          case "opendocfinished":
+            if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("TESTING")) {
+              this.opendocfinished = true;
```

Comment:
Name this `this._openDocFinished` to use same casing as the other member variables and to indicate that it's private, especially since this is only used for testing.

Moreover, since I agree with @Snuffleupagus that this is a bit hacky (although more solid than fixed timeout that may or may not be enough), I'd at least like to see a short comment here explaining why this is there; something like:

`// Signal the integration tests that open actions have run.`